services/gateway/apis/http-version3/methods/modules/token/tokenAccountExists.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const tokenAccountExistsSource = require('../../../../../sources/version3/tokenAccountExists');
+const envelope = require('../../../../../sources/version3/mappings/stdEnvelope');
+const regex = require('../../../../../shared/regex');
+const { transformParams, response, getSwaggerDescription } = require('../../../../../shared/utils');
+
+module.exports = {
+	version: '2.0',
+	swaggerApiPath: '/tokens/account/exists',
+	rpcMethod: 'get.tokens.account.exists',
+	tags: ['Tokens'],
```

Comment:
Please update the tag to `Token` everywhere.
```suggestion
	tags: ['Token'],
```

Reply:
Applied.